miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListSubmissionsController.java

```diff
@@ -23,57 +23,29 @@
 
 package uk.ac.bbsrc.tgac.miso.webapp.controller;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
 import org.springframework.web.bind.annotation.ModelAttribute;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.servlet.ModelAndView;
 
-import uk.ac.bbsrc.tgac.miso.core.data.Submission;
-import uk.ac.bbsrc.tgac.miso.core.manager.RequestManager;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+import uk.ac.bbsrc.tgac.miso.service.SubmissionService;
+import uk.ac.bbsrc.tgac.miso.webapp.util.ListItemsPage;
 
-/**
- * com.eaglegenomics.miso.web
- * <p/>
- * Info
- * 
- * @author Rob Davey
- * @since 0.0.2
- */
 @Controller
 public class ListSubmissionsController {
-  protected static final Logger log = LoggerFactory.getLogger(ListSubmissionsController.class);
-
   @Autowired
-  private RequestManager requestManager;
+  private SubmissionService submissionService;
 
-  public void setRequestManager(RequestManager requestManager) {
-    this.requestManager = requestManager;
+  @RequestMapping("/submissions")
+  public ModelAndView listSubmissions(ModelMap model) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)